src/middlewares/commentReaction.js

```diff
@@ -1,5 +1,4 @@
-import models from '../api/models';
-import statusCodes from '../helpers/constants/status.codes';
+import ifreacted from '../helpers/commonAction/commentReaction';
```

Comment:
@truestbyheart the name`ifreacted` here needs some editing, I think

Reply:
Since am checking if the user has either liked or disliked the best can come up with is ``didReact`` to indicate the user has previously liked or disliked (Reacted).

Reply:
Can you share your thoughts on this? @MemunaHaruna